feedback/feedback.py

```diff
@@ -11,89 +11,104 @@
 import six
 
 from xblock.core import XBlock
-from xblock.fields import Scope, Integer, String, List, Float
+from xblock.fields import Boolean, Dict, Integer, List, Float, Scope, String
+from xblockutils.resources import ResourceLoader
 from web_fragments.fragment import Fragment
 
 
-# We provide default text which is designed to elicit student thought. We'd
+# Make '_' a no-op so we can scrape strings
+_ = lambda text: text
```

Comment:
Why do we need this function here? Can we just unwrap default values as it is and delete redundant function and wrapping?

Reply:
I use it to collect translations